### PR TITLE
2884 Add RM UI group setup scripts

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
We need some groups defined for day to day RM support and elevated write access roles for the support tool UI/r-ops.

# What has changed
- Add UI permissions scripts

# How to test?
Check the lists of permissions make sense, RM SUPPORT should be all the global read only permissions and RM ELEVATED should have write access but only on our non prototype features.

# Links
https://trello.com/c/ZxqCN8wb/2884-productionise-support-tool-set-up-groups-3

### CODE REVIEW

**Review by liamtoozer — APPROVED** [review]

**Review by NickGrantONS — APPROVED** [review]
I think we should remove all print and fulfilment activities, or include them all... not have some and not others.

Have tested though and these groups work great.

**Comment by AdamHawtin** [comment]
@NickGrantONS I've got rid of all permissions relating to print and or fulfilments 👍

**Review by NickGrantONS — APPROVED** [review]
I have retested since you removed the print/fulfilment permissions and they still work using the updated Support Tool: https://github.com/ONSdigital/ssdc-rm-support-tool/pull/100

File: ui-permissions/RM-Elevated-Permissions.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+INSERT INTO casev3.user_group (id, name) VALUES ('e9ed16cb-8589-435b-898b-38c038bcdaaf', 'RM ELEVATED');
+
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('d0ad4df3-ee5b-4761-bd24-0546969d836e', 'LIST_SURVEYS', 'e9ed16cb-8589-435b-898b-38c038bcdaaf', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('16d5d137-5cc4-4079-ac13-24cfdd82eb07', 'VIEW_SURVEY', 'e9ed16cb-8589-435b-898b-38c038bcdaaf', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('f1095161-a1e5-4b3d-abcf-40ea21cda12e', 'CREATE_SURVEY', 'e9ed16cb-8589-435b-898b-38c038bcdaaf', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('376e3c34-1b3f-464c-aaa0-f3bd77960563', 'CREATE_PRINT_TEMPLATE', 'e9ed16cb-8589-435b-898b-38c038bcdaaf', NULL);
```
Comment (NickGrantONS) [inline ui-permissions/RM-Elevated-Permissions.sql:6]: should remove this print permission

File: ui-permissions/RM-Support-Permissions.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+INSERT INTO casev3.user_group (id, name) VALUES ('b19a77bd-6a02-4851-8116-9e915738b700', 'RM SUPPORT');
+
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('cc542d88-b1e6-4ab0-ba09-9857a4cc5877', 'LIST_SURVEYS', 'b19a77bd-6a02-4851-8116-9e915738b700', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('18c78ba8-17ec-4c09-972d-c1c6c88359d5', 'VIEW_SURVEY', 'b19a77bd-6a02-4851-8116-9e915738b700', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('f1f526ee-a10e-4772-923b-ec3586826cbf', 'LIST_COLLECTION_EXERCISES', 'b19a77bd-6a02-4851-8116-9e915738b700', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('28e30f2e-2d1b-4fa1-92ca-cd8323179c55', 'VIEW_COLLECTION_EXERCISE', 'b19a77bd-6a02-4851-8116-9e915738b700', NULL);
+INSERT INTO casev3.user_group_permission (id, authorised_activity, group_id, survey_id) VALUES ('fe549c89-961a-4191-b359-636143dc4c50', 'LIST_ALLOWED_PRINT_TEMPLATES_ON_ACTION_RULES', 'b19a77bd-6a02-4851-8116-9e915738b700', NULL);
```
Comment (NickGrantONS) [inline ui-permissions/RM-Support-Permissions.sql:7]: should remove this print permission

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
oops spoke too soon... 2 print permissions to remove

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
I had to remove `LIST_ALLOWED_PRINT_TEMPLATES_ON_ACTION_RULES` but it's all working just fine

**Review by NickGrantONS — APPROVED** [review]
Tested it and it works great!

**Pull request merged**